caffe2/python/operator_test/activation_ops_test.py

```diff
@@ -215,3 +217,27 @@ def leaky_relu_ref(X):
         self.assertReferenceChecks(gc, op, [X], leaky_relu_ref)
         # Check over multiple devices
         self.assertDeviceChecks(dc, op, [X], [0])
+
+    @given(X=hu.tensor(),
+           fast_gelu=st.booleans(),
+           **hu.gcs)
+    def test_gelu(self, X, fast_gelu, gc, dc):
+        op = core.CreateOperator(
+            "Gelu",
+            ["X"],
+            ["Y"],
+            fast_gelu=fast_gelu,
+        )
+
+        def gelu_ref(X):
+            cdf = 0.5 * (1.0 + erf(X / np.sqrt(2.0)))
+            return (X * cdf,)
+
+        tol = 1e-3 if fast_gelu else 1e-4
```

Comment:
do we want to try 1e-5 tolerance in non fast case.

Reply:
The default value is 1e-4
https://github.com/pytorch/pytorch/blob/master/caffe2/python/hypothesis_test_util.py#L563

Reply:
make sense. My concern is unnecessary. :-)